client/asset/btc/btc.go

```diff
@@ -1885,12 +1885,15 @@ func (btc *ExchangeWallet) AuditContract(coinID, contract, txData dex.Bytes, sin
 			contractHash, addr.ScriptAddress())
 	}
 
-	// Broadcast the transaction.
-	if hashSent, err := btc.node.sendRawTransaction(tx); err != nil {
-		btc.log.Debugf("Rebroadcasting counterparty contract %v (THIS MAY BE NORMAL): %v", txHash, err)
-	} else if !hashSent.IsEqual(txHash) {
-		btc.log.Errorf("Counterparty contract %v was rebroadcast as %v!", txHash, hashSent)
-	}
+	// Broadcast the transaction, but do not block because this is not required
+	// and does not affect the audit result.
+	go func() {
+		if hashSent, err := btc.node.sendRawTransaction(tx); err != nil {
+			btc.log.Debugf("Rebroadcasting counterparty contract %v (THIS MAY BE NORMAL): %v", txHash, err)
+		} else if !hashSent.IsEqual(txHash) {
+			btc.log.Errorf("Counterparty contract %v was rebroadcast as %v!", txHash, hashSent)
+		}
+	}()
```

Comment:
imo it would be best to also add a timeout and give up and log if it's something absurd.

Reply:
I agree with @JoeGruffins in spirit, but it isn't straightforward to get a custom `context.Context` into `call`, so you'd have to write a custom call handler or something. Since it's really just for messaging, it's hard to justify the effort.